### DISCUSSION TITLE
Mark file as safe (SmartScreen warning)

### DISCUSSION THREAD

**Discussion by magliocchetti** [timschneeb/GalaxyBudsClient#206] [General]
Regarding the SmartScreen warning, when you download the file it asks you if you want to keep it anyway. Near that option there is also another one to "mark the file as safe", that will open a Microsoft SmartScreen feedback page to send the report. In that page you should choose "I think this app is safe". I think that anyone who downloads the Windows installer should spend some seconds doing this so they'll hopefully remove that warning.

NB: Also, in that page there is another option for the developer of the app with which he can certify that the file is safe, but I don't know if that's enough or if he needs some sort of paid certificate to send to Microsoft.

**Comment by timschneeb** [comment]
Hey, yes this is true.
Also, I could get a paid certificate. Unfortunately, even the most basic certificate costs already 95 bucks per year and is not _that_ affordable for a free and open-source project. I'm considering getting one nonetheless soon.
